README.md

```diff
@@ -70,21 +70,17 @@ not "drain" from the simulation.
 {
   "nodes": [
     {
-      "LND": {
-        "id": "Alice",
-        "address": "https://localhost:10011",
-        "macaroon": "/path/admin.macaroon",
-        "cert": "/path/tls.cert"
-      }
+      "id": "Alice",
```

Comment:
I think that we need to expand the readme with an explanation of these different formats for CLN or LND - this isn't very easy for newcomers to understand. 

My instinct would be to have a `### Setup - Node Connection`  section that shows what it looks like for CLN and LND?

Reply:
I agree. Should we do this here or in a followup?

Reply:
I think it's generally a good idea to keep code changes in the same PR as their documentation so that we don't forget (and then docs get out of sync).

Reply:
And in this case, needs a rebase in the readme anyway so let's def do it in the same PR.